src/Types.jl

```diff
@@ -336,6 +336,7 @@ Attributes:
 immutable SkyPatch
     center::Vector{Float64}
     radius::Float64
```

Comment:
Yes, will do. Similarly, `initialize_objid` is no longer used. Can we delete it?